projects/wasmtime/Dockerfile

```diff
@@ -15,7 +15,13 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-builder
-RUN apt-get update && apt-get install -y make autoconf automake libtool curl cmake python llvm-dev libclang-dev clang ocaml-nox
+RUN apt-get update && apt-get install -y make autoconf automake libtool curl cmake python llvm-dev libclang-dev clang
+
+# Install a newer version of OCaml than provided by Ubuntu 16.04 (base version for this image)
+RUN echo | sh <(curl -sL https://raw.githubusercontent.com/ocaml/opam/master/shell/install.sh) && \
+  opam init --disable-sandboxing --yes && \
+  opam install ocamlbuild --yes && \
+  eval $(opam env)
```

Comment:
@alexcrichton, this last command updates the environment; I can't remember exactly if Docker will maintain this environment for subsequent `RUN` commands but I suspect not. Is there a way to try out a build and if it fails, move this to, e.g., the `build.sh`?

Reply:
Nah this `eval` won't be present for the build later on. I think moving this to the `build.sh` in this repo should be fine though